Adapt Balancer scripts to Aragon Liquidity rewards program (#1)
aragonone/ant-balancer-rewards: README.md

> <h1 align=center><code>ANT Mining</code></h1>

Reviewer: ```suggestion
<h1 align=center><code>ANT Liquidity Rewards (Balancer)</code></h1>
```

> Set of scripts to calculate ANT liquidity mining distributions

Reviewer: ```suggestion
Set of scripts to calculate ANT liquidity rewards distributions. See [announcement blogpost](https://aragon.org/blog/ant-liquidity-rewards-now-on-balancer).
```

## Usage

```
node index.js --period 1 [--config ./test/config_mainnet.js] [--output reports]
```

This will run run all historical calculations by block. Using an infura endpoint this may take upwards of TODO hours. For a local archive node, the sync time is roughly TODO minutes. Progress bars with estimates are shown during the sync. Reports will be saved in the folder for the given week specified

```
node sum.js --period 1 [--config ./test/config_mainnet.js] [--output reports]
```

After all reports are generated, `sum.js` will create a final tally of user address to ANT received. This is stored in the report week folder at `_totals.json`

## Distributions (TODO)

> 50,000 ANT will be distributed directly to addresses after the rewards program is over. Due to block gas limits, the tx's to batch transfer ANT will need to be split up across blocks. In order to prevent favoring certain accounts, the block hash of the `endBlock` will be the starting point and addresses will be ordered alphabetically for distributions.

Reviewer: ```suggestion
25,000 ANT will be distributed directly to addresses after the rewards program is over. Due to block gas limits, the tx's to batch transfer ANT will need to be split up across blocks. In order to prevent favoring certain accounts, the block hash of the `endBlock` will be the starting point and addresses will be ordered alphabetically for distributions.
```
